activation_logger.go

```diff
@@ -84,18 +84,34 @@ func NewActivation(config ActivationLoggerConfig) (Logger, error) {
 	return l, nil
 }
 
+func (l *activationLogger) Debug(ctx context.Context, message string) {
+	l.LogCtx(ctx, "level", "debug", "message", message)
+}
+
 func (l *activationLogger) Debugf(ctx context.Context, format string, params ...interface{}) {
-	l.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf(format, params...))
+	l.Debug(ctx, fmt.Sprintf(format, params...))
 }
 
-func (l *activationLogger) Errorf(ctx context.Context, err error, format string, params ...interface{}) {
+func (l *activationLogger) Error(ctx context.Context, err error, message string) {
 	if err != nil {
-		l.LogCtx(ctx, "level", "error", "message", fmt.Sprintf(format, params...), "stack", microerror.JSON(err))
+		l.LogCtx(ctx, "level", "error", "message", message, "stack", microerror.JSON(err))
 	} else {
-		l.LogCtx(ctx, "level", "error", "message", fmt.Sprintf(format, params...))
+		l.LogCtx(ctx, "level", "error", "message", message)
 	}
 }
 
+func (l *activationLogger) Errorf(ctx context.Context, err error, format string, params ...interface{}) {
+	l.Error(ctx, err, fmt.Sprintf(format, params...))
+}
+
+func (l *activationLogger) Info(ctx context.Context, message string) {
+	l.LogCtx(ctx, "level", "debug", "message", message)
+}
+
+func (l *activationLogger) Infof(ctx context.Context, format string, params ...interface{}) {
+	l.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf(format, params...))
```

Comment:
Feels weird to have `level=debug` when using `Info*` methods.

Reply:
Oh, that's a copy-and-paste mistake :)

Reply:
Fixed.